src/script/components/MessagesList/Message/ContentMessage/asset/MessageButton/MessageButton.tsx

```diff
@@ -46,27 +45,20 @@ export const MessageButton = ({id, label, message, onClick = noop}: MessageButto
 
   return (
     <>
-      <button
-        type="button"
-        className={classNames('message-button', {
-          'message-button--selected': isSelected,
-        })}
+      <Button
+        variant={ButtonVariant.SECONDARY}
         onClick={onClick}
         data-uie-name={label}
         data-uie-uid={id}
         data-uie-selected={isSelected}
         data-uie-waiting={isWaiting}
+        showLoading={isWaiting}
+        isActive={isSelected}
+        loadingColor={COLOR.GRAY}
```

Comment:
mock look likes it should be just `var(--main-color)` no? so empty should be fine
![image](https://github.com/wireapp/wire-webapp/assets/78490891/1fe7e671-f35e-4117-9913-ba5eca543a99)

If it needs to be gray, some css variable that work in dark and light mode like `var(--text-input-placeholder)` would probably be more adequate, static colors are the evil